portfolio/src/main/java/com/google/sps/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.google.sps;
+
+//Class to store constants
+public final class Constants {
+    
+  public static final String COMM  = "Comm";
```

Comment:
It will be better, if your variables are descriptive.
I dont understand the difference between COMM and COMMENT

Reply:
Got it. Will do the changes!

Reply:
[deployed link](https://achoudhary-sps-summer20.df.r.appspot.com)
[comment store](https://achoudhary-sps-summer20.df.r.appspot.com/comments)